File: IR LLVM/Micro02/Micro02.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int num1;
    int num2;

    printf("Digite o primeiro numero: ");
    scanf("%d", &num1);
    printf("Digite o segundo numero: ");
    scanf("%d", &num2);
    if(num1 > num2)
        printf("O primeiro numero %d é maior que o segundo %d\n", num1, num2);
    else
        printf("O segundo numero %d é maior que o primeiro %d\n", num2, num1);
    return 0;
}

File: PASCAL/Micro02.pas
program micro02;

var num1, num2: Integer;

begin
    write('Digite o primeiro numero: ');
    readln(num1);
    write('Digite o segundo numero: ');
    readln(num2);
    if (num1 > num2) then
        begin
            write('O primeiro numero, ');
            write(num1);
            write(' e maior que o segundo numero, ');
            writeln(num2);
        end
    else
        begin
            write('O segundo numero, ');
            write(num2);
            write(' e maior que o primeiro numero, ');
            writeln(num2);
        end;
end.